lib/ipmi_hpmfwupg.c

```diff
@@ -1417,9 +1416,13 @@ HpmfwupgGetBufferFromFile(char *imageFilename,
 				"Failed to read file %s size %d", 
 				imageFilename,
 				pFwupgCtx->imageSize);
-		rc = HPMFWUPG_ERROR;
-	}
+		goto ret_close;
+	} else
+	    rc = HPMFWUPG_SUCCESS;
```

Comment:
This indentation is wrong (mixed tabs/spaces). Besides, this whole `else` is not needed here.